File: Deafios em C#/1066 - Pares, Ímpares, Positivos e Negativos.cs
using System; 

//Developed by: @LucasMarcuzo
class URI1066 {
    static void Main(string[] args) { 

        int w, x, y, z;
        w=x=y=z=0;

        int[] vetor=new int[5];

        for(int i=0; i<vetor.Length; i++)
        {
           vetor[i]=int.Parse(Console.ReadLine());
        }

        for(int i=0; i<vetor.Length; i++)
        {
            if(vetor[i]>0){
                w++;
            }if(vetor[i]<0){
                x++;
            }if(vetor[i] % 2 == 0){
                y++;
            }if(vetor[i] % 2 != 0){
                z++;
            }
        }

        Console.WriteLine("{0} valor(es) par(es)",y);
        Console.WriteLine("{0} valor(es) impar(es)",z);
        Console.WriteLine("{0} valor(es) positivo(s)",w);
        Console.WriteLine("{0} valor(es) negativo(s)",x);

    }

}

File: Desafios em JavaScript/1066 - Pares, Ímpares, Positivos e Negativos.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');

var w=0, x=0, y=0, z=0;
var vetor = new Array(5);

for(let i=0; i<vetor.length; i++)
{
    vetor[i]=parseInt(lines.shift());
}

for(let i=0; i<vetor.length; i++)
{
    if(vetor[i]>0){
        w++;
    }if(vetor[i]<0){
        x++;
    }if(vetor[i] % 2 === 0){
        y++;
    }if(vetor[i] % 2 !== 0){
        z++;
    }
}

console.log(`${y} valor(es) par(es)`);
console.log(`${z} valor(es) impar(es)`);
console.log(`${w} valor(es) positivo(s)`);
console.log(`${x} valor(es) negativo(s)`);